validphys2/src/validphys/cuts/filters.yaml

```diff
@@ -388,3 +388,8 @@
   FNS: FONLL-C
   IC: True
   rule: "Q2 > 8"
+
+- dataset: H1_1JET_319GEV_290PB-1_DIF_PTQ2
+  reason: |
+    There are 8 Q^2 bins, however the grid for the 1st bin is unavailable.
+  rule: "Q2 >= 8"
```

Comment:
Please move this a bit up in the file to where the dataset cuts are.

Reply:
which line exactly?

Reply:
The exact line doesn't matter, but right now this is together with the `process_type'` while all datasets are at the top.